### ISSUE TITLE
Add new machine learning algorithms

### ISSUE THREAD

**Issue by aswinzz** [aswinzz/Machine-Learning-Algorithms#1]
refer readme.md on how to add new algorithms

**Comment by veshitala** [comment]
Can i add following algorithms in a jupyter notebook 
1.Clustering-KMeans
2.logistic regression
3.Linear Regression
4.Random Forest

**Comment by MayV** [comment]
Please review and merge pr #8

**Comment by MayV** [comment]
Can I add more machine learning algorithms ?

**Comment by aswinzz** [comment]
@MayV  yeah sure you can add as many as you want , but make sure nothing is repeated

**Comment by veshitala** [comment]
> @MayV yeah sure you can add as many as you want , but make sure nothing is repeated

I added Some Algorithm From Scratch Please review .

**Comment by veshitala** [comment]
@aswinzz  i have added some algorithm .Please review them

**Comment by shivamgupta077** [comment]
Please review and merge pr #8

**Comment by kss682** [comment]
#20 - implemented HebbNet algorithm for AND gate .

**Comment by harsh-surya** [comment]
@aswinzz implemented LSTM deep learning model

**Comment by AaahAahAha** [comment]
@aswinzz  used NEAT Algorithm to play Flappy Bird
